sdk/core/Azure.Core.Amqp/src/AmqpAddress.cs

```diff
@@ -86,13 +86,13 @@ public override bool Equals(object obj)
         public bool Equals(string other) =>
             _address == other;
 
-        /// <inheritdoc/>
+        /// <summary>Compares two cref="AmqpAddress" values for equality<summary/>
```

Comment:
Missing some XML. Script may have a problem?
```suggestion
        /// <summary>Compares two <see cref="AmqpAddress"/> values for equality<summary/>
```